helperAPI.py

```diff
@@ -538,9 +538,17 @@ async def processTasks(message):
         "Authorization": f"Bot {DISCORD_TOKEN}",
         "Content-Type": "application/json",
     }
-    PAYLOAD = {
-        "content": message,
-    }
+
```

Comment:
This can all be simplified to:
```python
PAYLOAD = {
        "content": "" if embed else message,
        "embeds": [message] if embed else [],
}
```